xbmc/utils/EGLUtils.cpp

```diff
@@ -667,13 +667,16 @@ void CEGLContextUtils::SetDamagedRegions(const CDirtyRegionList& dirtyRegions)
   }
   else
   {
-    EGLint height = eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height);
+    EGLint height;
+    eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height);
```

Comment:
If eglQuerySurface would fail:  EGL_FALSE is returned on failure, EGL_TRUE otherwise. value is not modified when EGL_FALSE is returned. 

So - needs a fallback or are we sure it will always return EGL_TRUE?

Reply:
I'd say we can just assume that it succeeds. But we should still zero initialize `height` so the behavior is consistent in the very unlikely case something goes wrong.

Reply:
I moved the query to the surface init, and cache the value.